content/en/blog/2024/kubecon-eu.md

```diff
@@ -0,0 +1,130 @@
+  by Juraci Paixão Kröhling, Grafana Labs & Alexandre Magno Prado Machado, Pismo
+  Soluções Tecnológicas<br> Tuesday, March 19th • 15:15 - 15:40
+- **[⚡ Lightning Talk: Not Just Enterprise. Modern Java App CI/CD Observability with OTel, Quarkus and Gradle](https://sched.co/1YFin)**<br>
+  by Oleg Nenashev, WireMock<br> Tuesday, March 19th • 15:45 - 15:50
+- **[Shift Into an Observability Mindset with OpenTelemetry](https://sched.co/1YFjB)**<br>
+  by Daniel Gomez Blanco, Skyscanner<br> Tuesday, March 19th • 15:45 - 16:15
+- **[⚡ Lightning Talk: Federated Search Over Distributed Observability Data](https://sched.co/1YFjC)**<br>
+  by Kalyan Kolachala, Intuit<br> Tuesday, March 19th • 15:55 - 16:00
+- **[⚡ Lightning Talk: Application Security Through the Lens of OpenTelemetry - Yosef Arbiv, Outshift by Cisco](https://sched.co/1YFf5)**<br>
+  by Kalyan Kolachala, Intuit<br> Tuesday, March 19th • 16:05 - 16:10
+- **[Lazy Robots: Telemetry Buffering on Android](https://sched.co/1YFk3)**<br>
+  by Cesar Munoz, Elastic & Jason Plumb, Splunk<br> Tuesday, March 19th •
+  17:00 - 17:25
+- **[OpAMP in Action: User Configurable Observability Pipelines](https://sched.co/1YFk6)**<br>
+  by Srikanth Chekuri, SigNoz<br> Tuesday, March 19th • 17:00 - 17:25
+
+{{% alert title="Important access note" color="danger" %}}
+
+You need an _in-person all-access_ pass for on-site access to **Observability
+Day**. For details, see [KubeCon registration][]. If you have a virtual ticket,
+you will be able to follow **Observability Day** through a live stream.
+
+{{% /alert %}}
+
+## OpenTelemetry Project Booth
+
+Drop by and say _"Hi!"_ at the OpenTelemetry project booth in the KubeCon NA
+Project Pavilion. If you’re lucky, you may even pick up some OpenTelemetry swag.
+
+[Pavilion hours](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/program/project-opportunities/)
+for all projects during KubeCon are as follows:
+
+- Wednesday, March 20th • 10:45 - 20:00 (includes KubeCrawl + Cloud NativeFest
+  from 18:00 - 20:00)
+- Thursday, March 21th • 10:30 - 17:00
+- Friday, March 22th • 10:30 - 14:30
+
+_Please note, that we will use the flexibility to choose our own hosting hours.
+This is only a general guidance._
+
+You can help us improve the project by sharing your thoughts and feedback about
+your OpenTelemetry adoption, implementation, and usage. We will create action
+items from your comments as appropriate. Check [#otel-user-research][] in CNCF's
+Slack instance for results and action item updates to come after KubeCon EU.
+
+## OpenTelemetry Observatory
+
+In addition to our booth at the Project Pavilion, please visit the OpenTelemetry
+Observatory presented by Splunk in the Expo Hall! This will be a place for
+informal chats, meetups, and other discussions led by OpenTelemetry community
+members and maintainers. If you’d like to participate and lead a discussion or
```

Comment:
@svrnm we'll actually be posting a schedule and sign-ups for the various sessions. We can link to these directly from the blog post.

Reply:
awesome, share the links with me when you have them. We can also add them in after publishing the blog post

Reply:
Perfect! Hope to have somthing to you this week!